### PR TITLE
[Search-Bar] Fix placeholder issue

### LINKED ISSUE #1137: [Search-bar] The input placeholder is shown even if the query is not empty
# Current behaviour

Even if there is a 'query' the placeholder is shown.

image

# Expected behaviour

Hide the placeholder if there is a query.

## Versions

Focus-core: 0.14.0
Focus-components: 0.14.1

### PR DESCRIPTION
**Pull request by GuenoleK**
### Description

The placeholder stayed when the input value was already set and when the component was rendered.
1193f8fc-115d-11e6-8a17-5b151da3f3c8

### Patch

The placeholder was not updated. It's a material design lite input problem
We have to use the MaterialTextfield.change() function to update the component's placeholder correctly.

```
componentWillUpdate() { 
        this.refs.inputText.MaterialTextfield.change(this.props.value);
}
```

> Fixes #1137

### CODE REVIEW

File: src/components/input/text/index.js [hunk 1]
```diff
@@ -45,6 +45,11 @@ class InputText extends Component {
         const domEl = ReactDOM.findDOMNode(this.refs.htmlInput);
         return unformatter(domEl.value, MODE);
     };
+
+    componentDidUpdate() {
+        this.refs.inputText.MaterialTextfield.change(this.props.value);
+    }
```
Comment (pierr) [inline src/components/input/text/index.js]: @GuenoleK why isn't it handled by the material behaviour.

Comment (GuenoleK) [inline src/components/input/text/index.js]: Yeah you're right. Thank you

File: src/behaviours/material/index.js [hunk 2]
```diff
@@ -33,6 +33,12 @@ const Material = (ref, jsClass, watchedProp) => Component => class MaterialCompo
             Component.prototype.componentWillReceiveProps.call(this, nextProps);
         }
     }
+
+    componentDidUpdate() {
+        if(jsClass === 'MaterialTextfield') {
+            this.refs.inputText.MaterialTextfield.change(this.props.value);
```
Comment (pierr) [inline src/behaviours/material/index.js]: @GuenoleK It seems strange that there is only one class which triggers something.

**Comment by pierr** [comment]
@GuenoleK Can you check the build and rebase.
THanks!

**Comment by pierr** [comment]
@GuenoleK Can your rebase and push then we will work on making the test green again.

**Comment by TomGallon** [comment]
@pierr : as the error depends on MDL lib, are you ok to desactivate the test so we can merge the PR ?

**Pull request closed without merging**